src/views/forms/NewMarketForm/index.tsx

```diff
@@ -79,7 +79,12 @@ export const NewMarketForm = ({
       {
         key: 'deposit-lockup',
         label: stringGetter({ key: STRING_KEYS.DEPOSIT_LOCKUP }),
-        value: <Output type={OutputType.Percent} value={0.3256} />,
+        value: (
+          <Output
+            type={OutputType.Text}
+            value={`30${stringGetter({ key: STRING_KEYS.DAYS_ABBREVIATED })}`}
```

Comment:
i wonder if there's any language that goes like Days 30 :p

Reply:
Maybeeeee. This pattern is used in other areas of the web app as well

Reply:
If we get user reports we can fix for all cases